xgi/convert.py

```diff
@@ -87,6 +87,9 @@ def convert_to_hypergraph(data, create_using=None):
     ):
         from_incidence_matrix(data, create_using)
 
+    else:
+        raise XGIError("Not able to convert to hypergraph!")
```

Comment:
Suggestion: if possible, please add a reason why XGI failed rather than just saying that it failed.

Reply:
fixed.